velox/common/memory/SharedArbitrator.cpp

```diff
@@ -857,19 +875,18 @@ void SharedArbitrator::runGlobalArbitration() {
 
       // Check if we need to abort participant to reclaim used memory to
       // accelerate global arbitration.
-      //
-      // TODO: make the time based condition check configurable.
-      reclaimByAbort =
-          (getCurrentTimeMs() - startTimeMs) < maxArbitrationTimeMs_ / 2 &&
-          (reclaimByAbort || (allParticipantsReclaimed && reclaimedBytes == 0));
-      if (!reclaimByAbort) {
+      meetAbortCondition = (getCurrentTimeMs() - startTimeMs) >
+              maxArbitrationTimeMs_ * globalArbitrationAbortTimeRatio_ &&
+          (meetAbortCondition ||
+           (allParticipantsReclaimed && reclaimedBytes == 0));
+      if (globalArbitrationSkipSpill_ || meetAbortCondition) {
```

Comment:
How about just merge globalArbitrationSkipSpill_ into condition check above?
Also how about call it globalArbitrationWithoutSpill_

Reply:
I was separating them because it was easier to read. But yeah we can merge